build.sbt

```
@@ -23,6 +23,7 @@ lazy val certifiedMailSenderModuleName               = "certified-mail-sender"
 lazy val certifiedMailSenderModelsModuleName         = "certified-mail-sender-models"
 lazy val eservicesMonitoringExporterModuleName       = "eservices-monitoring-exporter"
 lazy val privacyNoticesUpdaterModuleName             = "privacy-notices-updater"
+lazy val eserviceVersionsArchiverModuleName          = "eservice-versions-archiver"
```

by convention we refer to eservices versions as `descriptors`.
I would keep this naming to avoid confusion (for us and others)
```suggestion
lazy val eserviceDescriptorsArchiverModuleName          = "eservice-descriptors-archiver"
```